bc_obps/registration/api/user_operator.py

```diff
@@ -293,6 +296,7 @@ def request_access(request, payload: SelectOperatorIn):
             if status != 200:
                 return status, message
 
+            # brianna how could one exist if this is an access request?
             # Making a draft UserOperator instance if one doesn't exist
```

Comment:
Not really related to the PR, but this endpoint is for requesting access. Users can't have have more than one request, so how could a previously existing UserOperator instance come about?

Reply:
It is possible for a user to have multiple records in the user_operator table in the scenario where:
They are declined access to an operator & then they request access to a different operator